contracts/terraswap_router/src/state.rs

```diff
@@ -7,6 +7,8 @@ use cw_storage_plus::Item;
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct Config {
     pub terraswap_factory: CanonicalAddr,
+    pub loop_factory: CanonicalAddr,
```

Comment:
do we need schema json update as well?

Reply:
https://github.com/terraswap/terraswap/pull/34/commits/7d56451c033fb07288eda67115e31a6742c76e00